kasa/cli.py

```diff
@@ -1245,13 +1239,16 @@ def _print_features(dev):
     feat = dev.features[name]
 
     if value is None:
-        unit = f" {feat.unit}" if feat.unit else ""
-        echo(f"{feat.name} ({name}): {feat.value}{unit}")
+        echo(f"Feature {name} has value {feat.value}")
```

Comment:
I'd keep the original one here, as it shows the id.

Reply:
Done